lib/full.css

```diff
@@ -624,3 +624,10 @@ sj-search-box sj-entity-suggestion.sj-search-box:last-of-type {
 ::shadow ::-webkit-scrollbar-thumb:active {
   background-color: rgba(255, 255, 255, 0.125);
 }
+.qp #material-one-middle .material-search::-webkit-input-placeholder{
+  color:<<FORE_PRIMARY>>;
+}
+.qp #material-one-middle .material-search, .qp #material-one-middle .material-search:focus {
+  background-color: rgba(255,255,255,0.16);
```

Comment:
![image](https://cloud.githubusercontent.com/assets/17199186/24323148/465aba28-1169-11e7-99fb-6a9a8afac745.png)
Uhm? It works.